index.bs

```diff
@@ -1477,7 +1486,9 @@ The [=authenticator data=] structure is a byte array of 37 bytes or more, as fol
         <td>
             Flags (bit 0 is the least significant bit):
             - Bit 0: [=Test of User Presence=] (`TUP`) result.
-            - Bits 1-5: Reserved for future use (`RFU`).
+            - Bit 1: Reserved for future use (`RFU1`).
```

Comment:
Can you explain a little why you aren't using Bit 1? I hold the position that the web spec shouldn't define the underlying protocol too specifically and this is an instance where the web spec is being more descriptive. Therefore if the reasoning here comes from the CTAP world, that's fine. But if there's discussion that leads to this justification, please do let us know. At least JC and I are far removed from the CTAP world.